### PR TITLE
RDF prefix validation script

### PR DESCRIPTION
**Pull request by hbcesar**
Including the script for checking RDF prefixes

### CODE REVIEW

**Comment by hbcesar** [comment]
Including the script that checks Shex files prefixes

File: scripts/run-actv-3-rdf.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#Checks if all declared prefixes are used in the RDF File
```
Comment (rajaram5) [inline scripts/run-actv-3-rdf.py]: Can you change the file name according this naming convention? https://github.com/naming-convention/naming-convention-guides/tree/master/python

File: scripts/run-actv-3-shex.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import glob
+import logging
+import sys
+
+# set log level
+logging.basicConfig(level=logging.INFO)
+
+root_path = "../"
+
+for filename in glob.iglob(root_path+ '**/*.rdf', recursive=True):
```
Comment (rajaram5) [inline scripts/run-actv-3-shex.py]: The file extension should be .shex

File: scripts/run-actv-3-shex.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import re
```
Comment (rajaram5) [inline scripts/run-actv-3-shex.py]: Can you change the file name according this naming convention? https://github.com/naming-convention/naming-convention-guides/tree/master/python

**Review by rajaram5 — CHANGES_REQUESTED** [review]
@hbcesar Really good work. I made minor comments. can you fix them?

**Review by rajaram5 — APPROVED** [review]
looks good

**Pull request merged**